zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/TransferListener.java

```diff
@@ -122,7 +120,7 @@ public void transferSucceeded(TransferEvent event) {
   @Override
   public void transferFailed(TransferEvent event) {
     transferCompleted(event);
-    event.getException().printStackTrace(out);
+    logger.warn(event.getException().getMessage());
```

Comment:
Warn and not error as it might be legit to not find a dependency in one repo (should fallback to the next)